### PR TITLE
Align the no-miss section to match the homepage

### PR DESCRIPTION
**Pull request by Peartes**
This commit addresses the issue [here](https://www.notion.so/curvelabs/339a249185e14ddcbd731e5ba07c46c7?v=f7cba822bb9043a5a87ab80b5e2b40ef&p=d9cf6ed33385477195f57f6d3514ad26)

### CODE REVIEW

File: src/home/home.scss [hunk 1]
```diff
@@ -127,7 +127,7 @@
       background-color: $BG02;
       height: 12em;
       align-content: center;
-      justify-content: space-around;
+      justify-content: space-between;
```
Comment (dkent600) [inline src/home/home.scss:130]: This does not result in what the wireframe is asking for.

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/home/home.scss (hunk 1 above)
Comment (Peartes) [reply src/home/home.scss:130]: Screenshot 2021-05-31 at 16 35 47

Here is the result of the change

Comment (dkent600) [reply src/home/home.scss:130]: take a thorough look  at the home page wireframe.  if you still don't see the problem, set up a zoom for us and you can show me.

**Comment by Peartes** [comment]
@dkent600  work on homepage would take care of this

**Pull request closed without merging**